roles/nginxplus/tasks/localhost.yml

```
@@ -8,7 +14,7 @@
     mode: 0755
   when: running_on_server
 
-- name: add pul svg file
+- name: nginxplus | add pul svg file
```

This is very nitpicky, but for clarity and ease of reading, I might change:
```suggestion
- name: nginxplus | add pul logo
```